numpy/core/src/umath/ufunc_object.c

```diff
@@ -4644,6 +4851,39 @@ PyUFunc_FromFuncAndDataAndSignature(PyUFuncGenericFunction *func, void **data,
                                      int unused, const char *signature)
 {
     PyUFuncObject *ufunc;
+    /*
+     * Use memcpy and an initialzed instance to set the newly allocated PyUFuncObject.
+       The version field is a constant and cannot be initialzed in C directly
```

Comment:
`*((int *)&ufunc.version) = UFUNC_VERSION` would work, I think

Reply:
While it would work, it is [undefined behaviour](https://stackoverflow.com/questions/3709548/changing-const-value-in-c) according to the C standard. So why even make it `const` in the first place, if we ourselves do not respect the difference?

Reply:
I doubt the `memcpy` is any better-defined.